### DISCUSSION TITLE
The topic about communication groups.

### DISCUSSION THREAD

**Discussion by qigeqigeqi** [apache/skywalking#10934] [General]
**I would like to know if there are more convenient communication groups apart from discussing on GitHub. I want to join a community where I can communicate better with others because relying solely on reading documentation and conducting research on my own is indeed quite inefficient.**

**Comment by Superskyyy** [comment]
There is, if you communicate in English, Slack group is available, otherwise in Chinese we have QQ group. See in the readme of this repo (bottom). @qigeqigeqi

**Reply by qigeqigeqi** [reply]
All right, thx.
